js/src/binary-client.ts

```diff
@@ -104,6 +111,16 @@ export interface RebuildResult {
   patternMatch: boolean;
 }
 
+/**
+ * The Result from calling the Dateilager Garbage Compiler
+ */
+export interface GCResult {
+  /**
+   *
+   */
```

Comment:
Either we include a comment about the interface property or not, but these empty lines definitely don't provide any value.